libs/common-services/src/test/java/uk/gov/di/ipv/core/library/fixtures/VcFixtures.java

```diff
@@ -434,70 +435,52 @@ static VerifiableCredential vcPassportMissingPassport() {
                         .build());
     }
 
+    private static VerifiableCredential generateAddressVc(TestVc.TestCredentialSubject subject) {
+        return generateVerifiableCredential(
+                TEST_SUBJECT,
+                Cri.ADDRESS,
+                TestVc.builder()
+                        .type(new String[] {VERIFIABLE_CREDENTIAL_TYPE, ADDRESS_CREDENTIAL_TYPE})
+                        .credentialSubject(subject)
+                        .evidence(null)
+                        .build(),
+                TEST_ISSUER_INTEGRATION,
+                Instant.ofEpochSecond(1658829720));
+    }
+
     VerifiableCredential VC_ADDRESS =
-            generateVerifiableCredential(
-                    TEST_SUBJECT,
-                    Cri.ADDRESS,
-                    TestVc.builder()
-                            .credentialSubject(
-                                    TestVc.TestCredentialSubject.builder()
-                                            .address(List.of(ADDRESS_1))
-                                            .build())
-                            .build());
+            generateAddressVc(
+                    TestVc.TestCredentialSubject.builder().address(List.of(ADDRESS_1)).build());
 
     static VerifiableCredential vcAddressTwo() {
```

Comment:
The mix of fields and static methods we have in this class confuses me sometimes. And can make the tests behaving in unexpected ways (timestamps and equalities)

Reply:
Yes, it's not quite as uniform as I'd like. If you'd like to address it, you could pick up https://govukverify.atlassian.net/browse/PYIC-6973 (possibly after the first wave of vocab refactoring!)